src/libraries/System.Console/tests/SyncTextWriter.cs

```diff
@@ -13,6 +13,7 @@
 public class SyncTextWriter
 {
     [Fact]
+    [SkipOnPlatform(TestPlatforms.Browser, "Browser bypasses SyncTextWriter for faster startup")]
```

Comment:
this should be still used and tested on MT build, right ?

Reply:
I suppose. How do we express that?

Reply:
`[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]`

Reply:
That presumes we disable it on all ST platforms instead of just ST wasm, right?

Reply:
are there any other ST platforms ?
we have 
`public static bool IsThreadingSupported => (!IsWasi && !IsBrowser) || IsWasmThreadingSupported;`